Sloth.Core/Domain/Transfer.cs

```diff
@@ -163,6 +163,8 @@ public string FullObjectToString()
             }
             return result;
         }
+
+        public string ShortFinancialSegmentString => $"{FinancialSegmentString?[0..13]}...";
```

Comment:
I've never seen this w/o using `string.substr()` but if this works it's ok to me.  clear enough what's happening.

Reply:
it takes the first 13 characters of the string. I'll add a comment